flow-osgi/src/main/java/com/vaadin/flow/osgi/support/OSGiVaadinInitialization.java

```diff
@@ -204,8 +186,23 @@ public void contextInitialized(ServletContextEvent event) {
 
         VaadinServletContext context = new VaadinServletContext(servletContext);
         // ensure the context is set into the context
```

Comment:
The comment is wrong I guess it should be `Ensure lookup is set into the context`
Also why does a getter set the 'default value' it should just return a value never set anything. Could you make a ticket about fixing this. Also the javadoc doesn't mention anything about setting attribute value on the getter.

Reply:
Comment is corrected.

https://github.com/vaadin/flow/issues/9517